docs/using-keystone-webhook-authenticator-and-authorizer.md

```diff
@@ -81,28 +74,10 @@ $ kubectl apply -f /etc/kubernetes/keystone-auth/policy-config.yaml
 ```
 Version 2:
 
+please refer to [configmap](../examples/webhook/keystone-policy-configmap.yaml) for the definition of 
```

Comment:
The keystone-policy-configmap.yaml doesn't contain the version 2 policy definition which is recommended. Please add the v2 content to that file and comment v1.